src/components/footer/Footer.js

```diff
@@ -1,7 +1,105 @@
-import React from "react";
+import React from 'react';
+import Nav from '../nav/Nav';
+import './Footer.scss';
 
 function Footer() {
-  return <div></div>;
+  return (
+    <div className="footer-Wrap">
+      <div className="footer-Aside">
+        <img
+          src={process.env.PUBLIC_URL + '/images/Footer/footerImg1.png'}
+          alt="footer-AsideImg"
```

Comment:
alt로 들어가는 값에는 Img, 이미지 등의 내용이 들어가지 않아도 됩니다!

Reply:
이미지의 alt값은 스크린리더기를 통해 시각장애인에 읽혀지기도 하고, 해당 이미지가 출력되지 않았을 때 어떤 이미지인지 텍스트로 알려주기 떄문에 명확하게 어떤 이미지인지 표기해주세요!

예를 들면 다음같이 할 수 있을 것 같네요.
```suggestion
          alt="인스타그램 로고"
```